.github/README-en-US.md

```diff
@@ -0,0 +1,44 @@
+**English (US)** | [Português (BR)](../README.md)
+
+<p align="center">
+  <img alt="Querido Diário" src="./images/querido-diario-logo.png" width="200">
+</p>
+
+# Documentation and Community
+
+General documentation related to *Querido Diário* and a dedicated space to [discussions](https://github.com/okfn-brasil/querido-diario-comunidade/discussions) and to the project community.
+
+*The documentation is still in progress.* :building_construction:
+
+# Contributing
+
+Thanks for considering contributing to *Querido Diário*! :tada:
+
+You can find how to do it in our [Contribution Guide](CONTRIBUTING-en-US.md).
+
+# *Querido Diário*
+
+Related repositories:
+- [Documentation and Community](https://github.com/okfn-brasil/querido-diario-comunidade)
+- [Scrapers](https://github.com/okfn-brasil/querido-diario)
+- [Toolbox](https://github.com/okfn-brasil/querido-diario-toolbox)
+- [Infrastructure](https://github.com/okfn-brasil/querido-diario-infra)
+- [Frontend](https://github.com/okfn-brasil/querido-diario-frontend)
+- [Data Processing](https://github.com/okfn-brasil/querido-diario-data-processing)
+- [API](https://github.com/okfn-brasil/querido-diario-api)
+- [API Wrapper](https://github.com/okfn-brasil/querido-diario-api-wrapper)
+- [Census](https://github.com/okfn-brasil/censo-querido-diario)
+- [Education (Pilot)](https://github.com/okfn-brasil/piloto-educacao)
```

Comment:
This project will probably be archived. We don't need to encourage contributions to it. Therefore, we can remove it